markdown/org/docs/designs/jane/instructions/en.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Jane Shift: Sewing Instructions"
+---
+
+You can sew your Jane entirely by machine, or by hand. But it is also possible to mix and match techniques to your liking. It is common practice for modern makers of historical clothing to sew the seams by machine but finish them by hand.
+
+Extant examples are stitched very finely to allow them to stand up to harsh laundering practices. You can stitch your seams with a backstitch and then finish them with a felling stitch.
+
+The shift Jane was based on is sewn with a technique called double felling. To do this you baste the edges of your seam back on both pattern pieces and then align them so both the wrong and right side show the folded over edge. Baste the seam in place and then fell the edge on the wrong and the right side down.
+
+
+<Note>
+Sewing in these instructions includes finishing your seam. So if you sew your sleeve gussets to your sleeves you want to also finish those seams before moving on.
+</Note>
+
+
+## Step 1
+Hem the neckline with the smallest hem you can manage
+
+## Step 2
+Construct the sleeve by sewing up the sleeve seam with space left for the sleeve gusset
+The sides to seam up are the ones that do not have a notch
+Repeat for the second sleeve
+
+## Step 3
+Add the sleeve gusset to your sleeves
+
+## Step 4
+Hem the sleeve ends
+
+## Step 5
+Sew the side  gores together
+
+## Step 6
+Mark the sideseam. This is the space left between the side gores and the sleeve.
+
+## Step 7
+Sew the sideseams
+
+## Step 8
+Set in the side gores
+
+## Step 9
+Set in the sleeves
```

Comment:
```suggestion
## Step 6: Set in the sleeves
Sew the sleeve to the body matching the notch to the shoulder. The sleeve should fit in the gap that has been left in the side seam. Make sure to not sew the sleeve shut when attaching.
Finish the seam.
Repeat for the other sleeve.

<Note>

You may find it easier to turn the body *wrong sides* out for this. It will depend on your construction method though.

</Note>
```